### ISSUE TITLE
Antergos : Gnome-Gmail won't open.

### ISSUE THREAD

**Issue by 1LinuxGuy** [davesteele/gnome-gmail#62]
On a fresh installation of Antergos (Gnome), I installed Gnome-Gmail from the AUR repo.

When trying to open Gnome-Gmail, all I get is a spining wheel... and it won't open.

What could be missing ?

Thanks.

**Comment by davesteele** [comment]
I'm not involved with AUR packaging, but...

The first time you run, it will put up a number of dialogs. One asks if you want gg to be the default mailer, another asks what emai account you want to use. After that, it will open an UAuth2 window to Google. You typically only see all of those once. After that, it will open a default browser window to your GMail page.

Are you sure the dialogs haven't been raised? 

You could try running "gnome-gmail" from the command line, and ^C after it has hung for a while. That will give an indication of where it is stuck.

**Comment by 1Claude** [comment]
I haven't seen any dialog window, like as usual.

From the Terminal, here is what I get 

Traceback (most recent call last):
  File "/usr/share/gnome-gmail/gnomegmail.py", line 928, in <module>
    main()
  File "/usr/share/gnome-gmail/gnomegmail.py", line 925, in main
    browser().open(gmailurl, new_browser, True)
  File "/usr/share/gnome-gmail/gnomegmail.py", line 134, in browser
    cmd.split(), universal_newlines=True).strip()
  File "/usr/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.6/subprocess.py", line 403, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.6/subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1333, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] Aucun fichier ou dossier de ce type: 'xdg-settings'

**Comment by davesteele** [comment]
So it can't find the xdg-settings command, which by the way is in the xdg-utils package.

You should file this bug with AUR, and reference this issue.

Expect problems if you are not running GNOME.

Also, I noticed that Arch says GNOME uses Wayland by default. Note that gnome-gmail won't currently work with Wayland. That will be fixed in a month or so.

**Comment by 1Claude** [comment]
I'm running on Gnome (Xorg)

The xdg-utils package was not installed. So, I installed it.

But I get the same result : it won't open nothing.

I tried to uninstall gnome-gmail, reboot, etc. No luck.

(I made sure to delete all the orphins packages in Pamac after uninstall it and reisntall.)

Now, when I run from the Terminal, I get no message. But it won't open either.

I guess some config file may have been left behind after uninstall and causing the problem.
